src/ResourceManagement/ServerManagement/ServerManagement/Generated/GatewayOperationsExtensions.cs

```diff
@@ -0,0 +1,738 @@
+
```

Comment:
AutoRest should generate headers by default. Please regen it after you wil lremove -Header NONE option in generate.cmd